server/src/App.ts

```diff
@@ -0,0 +1,65 @@
+import bodyParser from 'body-parser';
```

Comment:
please stick to the standard lowercase-with-dash naming that we've adopted elsewhere

Reply:
done